src/components/Card.svelte

```diff
@@ -15,23 +24,28 @@
 
 		return artistsOneline;
 	}
+	//when collapsed, pause audio and set expand indicator to false
```

Comment:
Typically, you don't want comments, and especially don't need it if the function naming is informative and the implementation is straight forward and understandable.